### PR TITLE
fix: footer ui

### PR DESCRIPTION
**Pull request by devdotmohit**
- **Refactor - refactored Get-solid-light, Press and footer component**
- **refactor - implmented the requested changes in footer, get-solid-light, press**
- **x**

### CODE REVIEW

File: src/atoms/buttons/primary/index.astro
Comment (tanishdotdev) [inline src/atoms/buttons/primary/index.astro:1]: Define this color in tailwind config file.

File: src/atoms/footer/footer-paragraph/grey/index.astro
Comment (tanishdotdev) [inline src/atoms/footer/footer-paragraph/grey/index.astro:1]: Define this color in tailwind config file.

File: src/components/common/footer/index.astro
Comment (tanishdotdev) [inline src/components/common/footer/index.astro:1]: 1. Define [#e2e2e2] in Tailwind config file.
2. Format the props in ascending order in the atoms in Line 76.

File: src/components/home/press/index.astro
Comment (tanishdotdev) [inline src/components/home/press/index.astro:1]: Format the props in ascending order in the atoms.

**Review by tanishdotdev — CHANGES_REQUESTED** [review]

File: src/components/common/footer/index.astro
Comment (devdotmohit) [reply src/components/common/footer/index.astro:1]: @tanishdotdev LogoIcon atom?

**Pull request closed without merging**